src/main/handlers.js

```diff
@@ -108,37 +106,37 @@ function installWin32 () {
     setProtocol()
 
     function setProtocol (err) {
-      if (err) log.error(err.message)
+      if (err) return log.error(err.message)
       protocolKey.set('', Registry.REG_SZ, name, setURLProtocol)
     }
 
     function setURLProtocol (err) {
-      if (err) log.error(err.message)
+      if (err) return log.error(err.message)
       protocolKey.set('URL Protocol', Registry.REG_SZ, '', setIcon)
     }
 
     function setIcon (err) {
-      if (err) log.error(err.message)
+      if (err) return log.error(err.message)
 
       const iconKey = new Registry({
         hive: Registry.HKCU,
-        key: '\\Software\\Classes\\' + protocol + '\\DefaultIcon'
+        key: `\\Software\\Classes\\${protocol}\\DefaultIcon`
```

Comment:
Have you tested this change? We don't have a test for this, so it should be manually tested before merging.